3.11/arangosearch.md

```diff
@@ -125,14 +138,21 @@ logical and comparison operators, as well as
 
 ### Understanding the Analyzer context
 
-ArangoSearch allows you to index the same field with multiple Analyzers.
+ArangoSearch Views allow you to index the same field with multiple Analyzers.
 This makes it necessary to select the right one in your query by setting the
 Analyzer context with the `ANALYZER()` function.
 
+{% hint 'tip' %}
+If you use Search Alias Views, you do not need to specify the Analyzer context
+with the `ANALYZER()` function in queries. The Analyzers are inferred from the
```

Comment:
It's not true. It allowed, for backward compatibility but if it will different from search-alias definition your query will be aborted with an error